resizable.js

```
@@ -414,6 +431,15 @@ proto.createHandle = function (handle, direction) {
 		//trigger callbacks
 		emit(self, 'resizeend');
 		emit(el, 'resizeend');
+
+		// persist inline styles if persist is configured
+		try {
+			const persist = (typeof self.persist === 'boolean') ? (self.persist ? 'session' : false) : self.persist;
+			if (persist && self.persistKey) {
+				const storage = (persist === 'local') ? localStorage : sessionStorage;
+				storage.setItem(self.persistKey, self.element.style.cssText);
+			}
+		} catch (_) {}
```

Not sure if silent fail is good here. Besides I don't see what can be source of error

---

Should I remove the catch?